packages/web-components/src/components/masthead/masthead-composite.ts

```diff
@@ -529,7 +528,7 @@ class DDSMastheadComposite extends LitElement {
    * The user authentication status.
    */
   @property({ attribute: 'user-status' })
-  userStatus?: USER_AUTHENTICATION_STATUS;
+  userStatus = 'Unauthenticated';
```

Comment:
Wondering `@carbon/ibmdotcom-services` still returns `Unauthenticated` string whereas it seems to return an user ID (email address) for authenticated state. Is it the case? If so, I suggest we define a const in https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.14.0/packages/services-store/src/types/profileAPI.ts so we don't need to define `'Unauthenticated'` string literals all over the place.